sci-libs/hdf5/hdf51102-1.10.2.recipe

```diff
@@ -0,0 +1,141 @@
+	cmd:h5format_convert$secondaryArchSuffix
+	cmd:h5diff$secondaryArchSuffix
+	cmd:h5dump$secondaryArchSuffix
+	cmd:h5fc$secondaryArchSuffix
+	cmd:h5import$secondaryArchSuffix
+	cmd:h5jam$secondaryArchSuffix
+	cmd:h5ls$secondaryArchSuffix
+	cmd:h5mkgrp$secondaryArchSuffix
+	cmd:h5perf_serial$secondaryArchSuffix
+	cmd:h5redeploy$secondaryArchSuffix
+	cmd:h5repack$secondaryArchSuffix
+	cmd:h5repart$secondaryArchSuffix
+	cmd:h5stat$secondaryArchSuffix
+	cmd:h5unjam$secondaryArchSuffix
+	cmd:h5watch$secondaryArchSuffix
+	lib:libhdf5$secondaryArchSuffix = 101.1.0 compat >= 101
+	lib:libhdf5_cpp$secondaryArchSuffix = 102.0.0 compat >= 102
+	lib:libhdf5_fortran$secondaryArchSuffix = 100.1.1 compat >= 100
+	lib:libhdf5_hl$secondaryArchSuffix = 100.1.0 compat >= 100
+	lib:libhdf5_hl_cpp$secondaryArchSuffix = 100.1.1 compat >= 100
+	lib:libhdf5hl_fortran$secondaryArchSuffix = 100.0.2 compat >= 100
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgettextpo$secondaryArchSuffix
+	lib:libgfortran$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	hdf51102${secondaryArchSuffix}_devel = $portVersion
+	devel:libhdf5$secondaryArchSuffix = 101.1.0 compat >= 101
+	devel:libhdf5_cpp$secondaryArchSuffix = 102.0.0 compat >= 102
+	devel:libhdf5_fortran$secondaryArchSuffix = 100.1.1 compat >= 100
+	devel:libhdf5_hl$secondaryArchSuffix = 100.1.0 compat >= 100
+	devel:libhdf5_hl_cpp$secondaryArchSuffix = 100.1.1 compat >= 100
+	devel:libhdf5hl_fortran$secondaryArchSuffix = 100.0.2 compat >= 100
+	"
+REQUIRES_devel="
+	hdf51102$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	hdf5${secondaryArchSuffix}_devel
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgettextlib$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gettext$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:grep
+	cmd:g++$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage hdf51102$secondaryArchSuffix \
+	"$libDir"/libhdf5.so.101.1.0 \
+	"$libDir"/libhdf5_cpp.so.102.0.0 \
+	"$libDir"/libhdf5_fortran.so.100.1.1 \
+	"$libDir"/libhdf5_hl.so.100.1.0 \
+	"$libDir"/libhdf5_hl_cpp.so.100.1.1 \
+	"$libDir"/libhdf5hl_fortran.so.100.0.2
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure ./configure \
+		--enable-hl \
+		--enable-build-mode=production \
+		--enable-symbols=yes \
+		--enable-fortran \
+		--enable-cxx \
+		--disable-sharedlib-rpath \
+		--disable-threadsafe
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install $jobArgs
+
+	rm $libDir/lib*.la || true
+
+	prepareInstalledDevelLibs \
+		libhdf5 \
+		libhdf5_cpp \
+		libhdf5_fortran \
+		libhdf5_hl \
+		libhdf5_hl_cpp \
+		libhdf5hl_fortran
+
+	fixPkgconfig
+
+	packageEntries devel \
+		$developDir
+
+	mkdir -p $docDir
+	mv $prefix/share/hdf5_examples $docDir/
```

Comment:
Examples should be in `developDocDir`, since you need the development package to run them.